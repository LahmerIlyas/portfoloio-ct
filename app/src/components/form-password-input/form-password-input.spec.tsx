import React from 'react';
import renderer from 'react-test-renderer';
import { Formik } from 'formik';
import { FormPasswordInput } from '.';

describe('<FormPasswordInput />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Formik initialValues={{}} onSubmit={() => {}}>
          <FormPasswordInput field="password" title="Password" />
        </Formik>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
