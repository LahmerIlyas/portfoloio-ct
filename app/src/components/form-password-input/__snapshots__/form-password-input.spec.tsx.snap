// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormPasswordInput /> renders correctly 1`] = `
<View
  style={
    Object {
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "black",
        "fontFamily": "Signika-Regular",
        "fontSize": 18,
      }
    }
  >
    Password
  </Text>
  <TextInput
    allowFontScaling={true}
    autoCapitalize="none"
    autoCompleteType="password"
    autoCorrect={false}
    onBlur={[Function]}
    onChangeText={[Function]}
    placeholder="Enter your password"
    placeholderTextColor="#F4F4F4"
    rejectResponderTermination={true}
    secureTextEntry={true}
    style={
      Object {
        "backgroundColor": "#F4F4F4",
        "borderRadius": 8,
        "color": "#666666",
        "fontFamily": "Signika-Regular",
        "fontSize": 16,
        "marginTop": 8,
        "paddingHorizontal": 24,
        "paddingVertical": 24,
      }
    }
    underlineColorAndroid="transparent"
  />
</View>
`;
